pkgs/by-name/pr/presenterm/package.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "presenterm";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "mfontanini";
+    repo = pname;
```

Comment:
```suggestion
    repo = "presenterm";
```

Reply:
@eclairevoyant - not sure on this suggestion, as it is by example from the [/languages-frameworks/rust.section.md](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/rust.section.md#buildrustpackage-compiling-rust-applications-with-cargo-compiling-rust-applications-with-cargo)

Reply:
example should be fixed then 😬

Reply:
This method is also used in the [23.05 manual](https://nixos.org/manual/nixpkgs/stable/#ssec-dart-applications)